ui/src/views/Quickstart/index.jsx

```diff
@@ -378,23 +397,30 @@ export default class QuickStart extends Component {
           </HelpView>
         }>
         <Fragment>
-          <div className={classes.orgRepo}>
-            <TextField
-              label="Organization Name"
-              name="owner"
-              onChange={this.handleOrgRepoChange}
-              value={owner}
-              autoFocus
-            />
-            <Typography className={classes.separator} variant="h5">
-              /
-            </Typography>
-            <TextField
-              label="Repository Name"
-              name="repo"
-              onChange={this.handleOrgRepoChange}
-              value={repo}
-            />
+          <div className={classes.orgRepoStatus}>
+            <List className={classes.listStyle}>
+              <ListItem>
+                <TextField
+                  label="Organization Name"
+                  name="owner"
+                  fullWidth
+                  onChange={this.handleOrgRepoChange}
+                  value={owner}
+                  autoFocus
+                />
+                <Typography className={classes.separator} variant="h5">
+                  /
+                </Typography>
+                <TextField
+                  label="Repository Name"
+                  name="repo"
+                  fullWidth
+                  onChange={this.handleOrgRepoChange}
+                  value={repo}
+                />
+              </ListItem>
+            </List>
```

Comment:
This list seems to only have one list item so probably doesn't make sense to put in a list. We can just use a div or something. Also, I noticed that when the spinner shows up (after filling both text fields), the width of the text fields change and so there's this sudden layout change. Let's try to fix that.